src/components/LoadingShape/styled/index.js

```diff
@@ -0,0 +1,119 @@
+import styled from 'styled-components';
+import attachThemeAttrs from '../../../styles/helpers/attachThemeAttrs';
+import AvatarIcon from '../icons/avatar';
+import ImageIcon from '../icons/image';
+
+const StyledShapeContainer = styled.div`
+    position: relative;
+    width: 100%;
+    height: 100%;
+`;
+
+const StyledImageIcon = styled(ImageIcon)`
+    ${props =>
+        props.shape === 'square' &&
+        `
+        width: 85%;
+        `}
+
+    ${props =>
+        props.shape === 'circle' &&
+        `
+        width: 60%;
+        `}
+
+    ${props =>
+        (props.shape === 'rect' || props.shape === 'rounded-rect') &&
+        `
+        max-width: 25%;
+        height: 80%;
+        `}
+`;
+
+const StyledAvatarIcon = styled(AvatarIcon)`
+    ${props =>
```

Comment:
I think we should add the color `backgroundMain` on the SVG

Reply:
Fixed